aw/ViewControllers/Reaches/Detail/ReachDetailContainerViewController.swift

```diff
@@ -0,0 +1,124 @@
+import CoreData
+import UIKit
+
+enum SubViews {
+    case detail, map
+}
+
+class ReachDetailContainerViewController: UIViewController {
+    @IBOutlet weak var detailView: UIView!
+    @IBOutlet weak var mapView: UIView!
+
+    var managedObjectContext: NSManagedObjectContext?
+    var reach: Reach?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        initialize()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        setupRightItems()
```

Comment:
What are "right" items? Maybe a different term or abstraction would be more appropriate here?